src/services/pcn-router/test/bridge/test2_2_4.sh

```diff
@@ -44,7 +44,7 @@ polycubectl router add r2 type=$TYPE
 polycubectl simplebridge add br1 type=$TYPE
 polycubectl simplebridge add br2 type=$TYPE
 
-router_add_port r1 to_br1 10.10.2.1 255.255.255.0
+router_add_port r1 to_br1 10.10.2.1 24
```

Comment:
Shouldn't be:
   router_add_port r1 to_br1 10.10.2.1/24
Without the "slash", the new syntax looks definitely not intuitive.
Btw, there was a bug in the CLI that prevented to handle the "/" in the proper way, but Mauricio fixed it some days ago.